index.html

```diff
@@ -0,0 +1,175 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="choose online tour for trip">
+  <link rel="stylesheet" href="./scss/bootstrap.scss">
+  <link rel="stylesheet" href="./styles/main.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&family=Nunito:wght@400;600;800&display=swap"
+    rel="stylesheet">
+  <title>Best Tour Plan</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="service-header">
+      <div class="container">
+        <div class="row top-header">
+          <div class="head-info col-12 col-lg-6">
+            <a class="head-info__tel-link" href="tel:+1234561500"><span class="head-info__phone">Questions? Need
+                Help?</span>
+              Call us on (+123) 456 1500</a>
+          </div>
+          <div class="col-12 col-sm-6 col-lg-3 top-header__link">
+            <a href="#" class="head-info__item">
+              <span class="icon icon-present gift_icon"></span>
+              <span>BTP Gift Card</span>
+            </a>
+          </div>
+          <div class="col-12 col-sm-6 col-lg-3 top-header__link">
+            <a href="#" class="head-info__item">
+              <span class="icon icon-star black_star"></span>
+              <span>Loyalty Program</span>
+            </a>
```

Comment:
Please, check class naming according to BEM.

Reply:
> Please, check class naming according to BEM.

classes renamed